src/devices/oregon_scientific.c

```diff
@@ -435,6 +436,20 @@ static int oregon_scientific_v2_1_parser(bitbuffer_t *bitbuffer) {
       //}
 
       return 1;
+	} else if (sensor_id == ID_UVR128 && num_valid_v2_bits==297) {
+		if ((validate_os_v2_message(msg, 297, num_valid_v2_bits, 12) == 0)) {
+		int uvidx = get_os_uv(msg, sensor_id);
+		data = data_make(
+		  "time",           "",           DATA_STRING, time_str,
+		  "model",          "",     	  DATA_STRING, "Oregon Scientific UVR128",
+		  "id",             "House Code", DATA_INT,    get_os_rollingcode(msg, sensor_id),
+		  "uv",             "UV Index",   DATA_FORMAT, "%u", DATA_INT, uvidx,
+		  "battery",        "Battery",    DATA_STRING, get_os_battery(msg, sensor_id)?"LOW":"OK",
```

Comment:
I think we concluded to use battery_low as generic name.

Reply:
yes, but it looks like this just a plain copy of all the other cases in the file. I'm not happy with "%u" either (uvidx is 0 to 165). I guess we should later change all of those "battery" instances with one commit.